lib/puma/binder.rb

```diff
@@ -251,7 +251,8 @@ def parse(binds, log_writer = nil, log_msg = 'Listening')
           else
             ios_len = @ios.length
             backlog = params.fetch('backlog', 1024).to_i
-            io = add_ssl_listener uri.host, uri.port, ctx, optimize_for_latency = true, backlog
+            opt = params.key?('low_latency') && params['low_latency'] != 'false'
```

Comment:
I think I would prefer:

```suggestion
low_latency = params['low_latency'] != 'false'
io = add_ssl_listener uri.host, uri.port, ctx, low_latency, backlog
```

Reply:
I can change the variable name from `opt to `low_latency`, but it is false by default, so the conditional should stay the same.  Also, the code added is the same as line 161, which is used with tcp connections:
https://github.com/puma/puma/blob/3f4e3c8ca689e39218459836b939c39fa81e2692/lib/puma/binder.rb#L161

Thoughts?

Reply:
Added a commit renaming the variable from `opt` to `low_latency`